### PR TITLE
Minor a11y fixes.

### PR DESCRIPTION
**Pull request by cahrens**
TNL-4112

@dianakhuang and @efischer19 can you guys review this small PR? I have Mark's blessing on it, and I re-ran acceptance and a11y tests on my sandbox.

@cptvitamin Here are the changes we discussed (without any tabindex changes). We can discuss tomorrow. My sandbox is updated for these changes:

https://cahrens.sandbox.edx.org/courses/course-v1:edx+ORA203+course/courseware/61944efb38a349edb140c762c7419b50/415c3ee1b7d04b58a1887a6fe82b31d6/

### CODE REVIEW

File: openassessment/xblock/static/sass/oa/utilities/_shame.scss [hunk 1]
```diff
@@ -189,10 +189,6 @@
     }
   }
 
-  .ui-staff__title {
```
Comment (cahrens) [inline openassessment/xblock/static/sass/oa/utilities/_shame.scss]: Note to reviewers: this bit of CSS is unnecessary. The same thing is being set in _staff.scss (without the !important), and it works just fine with this removed. I verified that ui-staff__title is only used in the places edited above.

**Comment by dianakhuang** [comment]
:+1: from me.

File: openassessment/templates/openassessmentblock/staff_area/oa_student_info.html [hunk 2]
```diff
@@ -13,13 +13,13 @@ <h2 class="title">
     </h2>
 
     <div class="staff-info__status ui-staff__content__section wrapper--ui--collapse staff-info__student__response">
-        <button class="ui-staff ui-slidable" aria-expanded="false" id="learner_response_{{ submission.uuid }}">
+        <button class="ui-staff ui-slidable" aria-controls="learner_response_content_{{ submission.uuid }}" aria-expanded="false" id="learner_response_{{ submission.uuid }}">
```
Comment (efischer19) [inline openassessment/templates/openassessmentblock/staff_area/oa_student_info.html]: Oh cool, I didn't even know `aria-controls` was a thing.

File: openassessment/xblock/static/sass/oa/utilities/_shame.scss (hunk 1 above)
Comment (efischer19) [inline openassessment/xblock/static/sass/oa/utilities/_shame.scss]: Hooray for removing unused CSS

**Comment by efischer19** [comment]
:+1: Looks great!

**Pull request merged**